docs/topics/patterns/token_id_webhook.md

```diff
@@ -0,0 +1,361 @@
+---
+sidebar_position: 6
+title: Leverage Momento Topics, webhooks, and disposable tokens in your event-driven architectures
+sidebar_label: Identifying unique resources
+description: Learn best practices around using Momento Topics, webhooks, and disposable tokens to process unique resources in your event-driven architectures.
+keywords:
+ - topics
+ - webhooks
+ - eda
+ - momento
+ - event-driven architecture
+---
+
+# Leverage Momento Topics, webhooks, and disposable tokens in your event-driven architectures
+
+Leveraging Momento Topics, webhooks, and disposable tokens enables secure and efficient data processing within event-driven systems. With Momento Topics, you can subscribe to messages on a topic and publish messages to a new topic. Webhooks serve as HTTP callbacks triggered in response to messages published to these topics, acting as stateless consumers. Disposable Tokens are designed to provide short-lived session tokens for users and services interacting with a system. By embedding unique identifiers (token_id) in each token, they ensure secure and traceable access to resources.
+
+The key here is that you can use Topics to facilitate real-time communication between users by granting them short-lived disposable tokens. These tokens can be embedded with user information (token_id), which, when messages are published, can be leveraged to access shared resources like Momento caches via webhook callbacks. For instance, you can personalize user experiences by accessing their information stored in a Momento cache by identifying the user through the token_id. Embedding information provides two significant advantages:
+- It enhances security and prevents a user from spoofing their identity.
+- It reduces data transfer costs as the user information is embedded with the token itself.
+
+![Architecture](@site/static/img/topics/patterns/token-id-webhook.png)
+
+## Pre-requisites
+1. A public facing endpoint to receive webhook events. This endpoint must accept POST requests and be able to receive inbound calls from Momento. More detail about the structure of this event is [described here](https://docs.momentohq.com/topics/webhooks/overview#example-event).
```

Comment:
The link can be a local reference, which i'd prefer to do in case we ever change the docs site for any reason